src/webui/www/private/scripts/prop-general.js

```diff
@@ -70,6 +70,7 @@ window.qBittorrent.PropGeneral ??= (() => {
         $("save_path").textContent = "";
         $("comment").textContent = "";
         $("private").textContent = "";
+        document.getElementById("progressPercentage").textContent = "";
```

Comment:
I want to make another PR to properly align columns in General tab - will move 'private' then if that's ok.

Reply:
This should move to the top as to follow the layout.

Reply:
Also `private` is at the wrong place. It should be moved to above `$("torrent_hash_v1").textContent = "";`

And `$("private").textContent = (data.has_metadata ...`  is placed wrong too.

I don't insisted this to be fixed now but you can help change it if you don't mind.